.ci/install_runtime.sh

```diff
@@ -57,17 +57,14 @@ NEW_RUNTIME_CONFIG="${PKGDEFAULTSDIR}/configuration.toml"
 clone_katacontainers_repo
 
 build_install_shim_v2(){
-	pushd "$runtime_src_path"
-	make
-	sudo -E PATH=$PATH make install
-	popd
+	build_static_artifact_and_install "shim-v2"
```

Comment:
Just to double check I understand this change (as I suspect the other commits will be along a similar pattern) - `build_static_artifact_and_install "shim-v2"` will trigger a bunch of scripts that will eventually get to [`kata-deploy-binaries.sh`](https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh) that will run `install_shimv2` and build the runtime binaries to `kata-deploy-binaries.sh:prefix` which is set to `readonly prefix="/opt/kata"` there and so we can just link to those built files here?

Reply:
That's exactly the case.

And the only binary we need to link (or add to the containerd path) is the `containerd-shim-kata-v2`.  All other binaries do not matter, as we use the full path in the specific `configuration.toml`, which is already using `/opt/kata/...`